src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -351,10 +351,7 @@ function CalendarWidget({
                             timezone={timezone}
                             currentlySelectedDate={currentlySelectedDate}
                             handleSelectDate={date => {
-                              if (
```

Comment:
Previously we only deselect the slot when the user clicks the current day. We're updating this to also deselect the slot when the user clicks a different day.